doofinder/includes/class-migration.php

```diff
@@ -97,6 +119,11 @@ private static function do_woocommerce_migration()
         return false;
     }
 
+    /**
+     * Default migration
+     *
+     * @return void
+     */
     private static function do_default_migration()
```

Comment:
Private and static and also, could we clean up this function please?

Reply:
I've removed this function as it is not necessary.